app/components/edit/contributor_component.html.erb

```diff
@@ -40,7 +40,8 @@
     <%# Contributor person name/orcid fields %>
     <%= render Elements::Forms::FieldsetComponent.new(label: 'Person', hidden_label: true, data: { contributors_target: 'personSection' }) do %>
       <%= render Elements::Forms::SelectFieldComponent.new(form:, field_name: :person_role, label_classes: 'fw-bold', tooltip: helpers.t('works.edit.fields.contributors.role.tooltip_html'),
-                                                           options: PERSON_ROLES, label: role_label, container_classes: 'mb-4') %>
+                                                           options: PERSON_ROLES, label: role_label, container_classes: 'mb-4',
+                                                           caption: helpers.t('works.edit.fields.contributors.role.definitions_link_label_html')) %>
```

Comment:
makes use of the new "caption" attribute on the component to pass in the html we need to show next to the label here